lib/config/common.ts

```diff
@@ -121,6 +121,7 @@ export interface RenovateConfig
   branchList?: string[];
   description?: string[];
 
+  platform?: string;
```

Comment:
I think this should be moved to `RenovateAdminConfig` interface

Reply:
`endpoint` also?

Reply:
Yes, i think so.

Reply:
✅ 8e94ce70c5da86c752adece72058849f7e72efb0